posthog/management/commands/execute_temporal_workflow.py

```diff
@@ -98,13 +104,14 @@ def handle(self, *args, **options):
         )
         retry_policy = RetryPolicy(maximum_attempts=int(options["max_attempts"]))
 
+        WORKFLOWS = BATCH_EXPORT_WORKFLOWS + DATA_IMPORT_WORKFLOWS + PROXY_SERVICE_WORKFLOWS
         try:
             workflow = next(workflow for workflow in WORKFLOWS if workflow.is_named(workflow_name))
-        except IndexError:
+        except StopIteration:
```

Comment:
Looks like this was changed from a `list` to a call to `next` here: https://github.com/PostHog/posthog/commit/a0fc0861750fbbf927227ffdbd32adfc1f170819#diff-cda328503ba79bae3715e89a66e64e3f7f8067d919500b953d811c6cc10d1d63. This is better but we didn't update the exception to a `StopIteration`.